src/ctapipe/image/cleaning.py

```diff
@@ -459,6 +494,129 @@ def time_constrained_clean(
     return mask_core | mask_boundary
 
 
+def lst_image_cleaning(
```

Comment:
General comment: can we name this something more descriptive than "LST" image cleaning? I would assume it is applicable to non-LST telescopes as well, so this name is confusing.

Reply:
I would have said so as well but all names I came up with  were extremely unwieldy, like (one of the shorter ones)

```
pedestal_tailcuts_with_time_cleaning
```

Reply:
time for an [acronym](https://github.com/bacook17/acronym)?
```
=====================================================
(   50) PELTATED  : PEdestaL TAilcuts with TimE Delta
(   43) PATASHTE  : PedestAl TAilcutS witH Time dElta
(   41) PALAMEDEA : PedestAL tAilcuts with tiME DEltA
(   40) PLASTID   : PedestaL tAilcutS with TIme Delta
(   40) PELTATE   : PEdestaL TAilcuts with Time dElta
(   39) PESTICIDE : PEdeSTal taIlCuts with tIme DElta
(   38) PLAITED   : PedestaL tAIlcuts with TimE Delta
(   38) PALATED   : PedestAL tAilcuts with TimE Delta
(   38) PLATTED   : PedestaL tAilcuTs with TimE Delta
(   36) PEETWEET  : PEdEsTal tailcuts With timE dElTa
(   36) PEDATED   : PEDestAl tailcuts with TimE Delta
(   36) PETALITE  : PEdesTAl taiLcuts wIth Time dElta
(   36) PETALLED  : PEdesTAL taiLcuts with timE Delta
(   35) PETTED    : PEdesTal tailcuts with TimE Delta
(   35) PLATED    : PedestaL tAilcuts with TimE Delta
(   33) PASTED    : PedestAl tailcutS with TimE Delta
(   33) PALSIED   : PedestAL tailcutS with tImE Delta
(   33) PATTEE    : PedestAl Tailcuts with TimE dElta
(   33) PETTISH   : PEdesTal TaIlcutS witH time delta
(   33) PELTAST   : PEdestaL TAilcutS with time delTa
(   33) PLUSHED   : PedestaL tailcUtS witH timE Delta
(   33) PEELITE   : PEdEstaL tailcuts wIth Time dElta
(   33) PALAITE   : PedestAL tAilcuts wIth Time dElta
(   33) PATHED    : PedestAl Tailcuts witH timE Delta
(   31) PITTED    : Pedestal taIlcuTs with TimE Delta
(   31) PETALED   : PEdesTAl taiLcuts with timE Delta
(   31) PITSIDE   : Pedestal taIlcuTS with tIme DElta
(   31) PALACED   : PedestAL tAilCuts with timE Delta
(   30) PETITE    : PEdesTal tailcuts wIth Time dElta
(   30) PELITE    : PEdestaL tailcuts wIth Time dElta
```
pesticide_cleaning?

Reply:
I'm not so sure you can pick letters in the middle pf a word for acronyms... 

But I'm ok with a longish name.  If I understand correctly this is tailcuts but using pedestal variance thresholds rather than fixed PE thresholds (which is nearly identical to what was used for the Whipple-10m since the early 1990s, though called _picture-boundary cleaning_ there) + it adds the time delta information.   I think in HESS we called this "NSB Cleaning" since it adapts to the the NSB level, but maybe there's a nicer name.

Reply:
i'll name it `NSBImageCleaner` and `nsb_image_cleaning` for now - i really have no idea for a better name. Maybe someone will come up with a better one.